microsoft-edge/devtools-guide-chromium/issues/index.md

```diff
@@ -20,118 +20,188 @@ keywords: microsoft edge, web development, f12 tools, devtools
+*  The accessibility-testing demo page has issues in the **Accessibility**, **Compatibility**, **Performance**, and **Security** categories.
+*  The SameSite cookies demo page has issues in the **Other** and **Security** categories.
+
+To open the issues tool:
+
+1.  Navigate to a webpage that contains issues to fix.  For example, open the [accessibility-testing demo page](https://microsoftedge.github.io/DevToolsSamples/a11y-testing/page-with-errors.html) in a new tab or window.
+
+    :::image type="complex" source="../media/issues-accessibility-demo-page-initial.msft.png" alt-text="Accessibility-testing demo page, which has some issues" lightbox="../media/issues-accessibility-demo-page-initial.msft.png":::
+       Accessibility-testing demo page, which has some issues
+    :::image-end:::
+
+    Or, open the [SameSite cookies demo page][GlitchSamesiteSandbox] in a new tab or window.  This page initially brings up a warning page, which reads `This site has been reported as unsafe`, `Hosted by samesite-sandbox.glitch.me`.  Select **More information** to expand the section, and then select **Continue to the unsafe site (not recommended)**.  We won't enter information in the demo page.
+
+    :::image type="complex" source="../media/samesite-demo-initial-page.msft.png" alt-text="Demo page for issues with SameSite cookies" lightbox="../media/samesite-demo-initial-page.msft.png":::
+       Demo page for issues with `SameSite` cookies
+    :::image-end:::
+
+1.  Select `F12` to open DevTools.  After a few seconds, the **Issues counter** \(![Issues counter](../media/issues-counter-icon.msft.png)\) appears, in the upper right corner of DevTools.  The number you get might be different, and the number might change when you refresh the page.
+
+    :::image type="complex" source="../media/samesite-demo-initial-page-welcome.msft.png" alt-text="SameSite cookie demo page with initial Welcome page and the Issues counter" lightbox="../media/samesite-demo-initial-page-welcome.msft.png":::
+       SameSite cookie demo page with initial Welcome page and the **Issues counter**
+    :::image-end:::
+
+1.  Select the **Issues counter** \(![Issues counter](../media/issues-counter-icon.msft.png)\).  The Issues tool opens.  For the [SameSite cookies demo page][GlitchSamesiteSandbox], issues in the **Other** and **Security** categories are listed.
+
+    :::image type="complex" source="../media/issues-tool-initial-view-samesite-page.msft.png" alt-text="The Issues tool for the SameSite demo page" lightbox="../media/issues-tool-initial-view-samesite-page.msft.png":::
+       The **Issues** tool for the SameSite demo page
+    :::image-end:::
+
+    For the [accessibility-testing demo page](https://microsoftedge.github.io/DevToolsSamples/a11y-testing/page-with-errors.html), issues in the **Accessibility** category are listed.  If you refresh the page, additional categories of issues are shown.
```

Comment:
I'm not sure why this is phrased this way, since all relevant categories should appear when page is first navigated to. If that's not happening, let's file a bug instead!

Proposed change: "Select the **Issues counter** \(![Issues counter](../media/issues-counter-icon.msft.png)\).  The **Issues** tool opens. For the accessibility-testing demo page, issues in several category are populated."